### PR TITLE
BUGZ-1522: Snap app on tablet is blocked when saving/canceling out of its Settings page - RC-84

### PR DESCRIPTION
**Pull request by danteruiz**
ticket - https://highfidelity.atlassian.net/browse/BUGZ-1522

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21429/HighFidelity-Beta-PR16227-9988e2c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21429/HighFidelity-Beta-Interface-PR16227-9988e2c.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21429/HighFidelity-Beta-PR16227-9988e2c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21429/HighFidelity-Quest-Beta-PR16227-9988e2c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21429/HighFidelity-Beta-PR16227-9988e2c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21429/HighFidelity-Beta-Interface-PR16227-9988e2c.exe

File: libraries/ui/src/ui/TabletScriptingInterface.h [hunk 1]
```diff
@@ -298,10 +298,26 @@ class TabletProxy : public QObject {
...
+     */
     Q_INVOKABLE void loadHTMLSourceImpl(const QString& url, const QString& injectedJavaScriptUrl, bool loadOtherBase, bool localSafeContext);
 
+     /**jsdoc
+     * Internal function, do not call from scripts
+     * @function TabletProxy#returnToPreviousAppImpl
+     */
+    Q_INVOKABLE void returnToPreviousAppImpl(bool localSafeContext);
+
+    Q_INVOKABLE void loadQMLOnTopImpl(const QVariant& path, bool localSafeContext);
```
Comment (hyperlogic) [inline libraries/ui/src/ui/TabletScriptingInterface.h]: I'm concerned that someone could call this via script and simply pass in true, to load local content.  Can you verify that this is not the case?

**Pull request merged**